### PR TITLE
Add helm chart

### PR DESCRIPTION
**Pull request by alexhumphreys**

### CODE REVIEW

File: deploy/helm/device-registry/values.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+service:
+  name: device-registry
+  type: ClusterIP
+  externalPort: 80
+  internalPort: 9001
+ingress:
+  enabled: false
+  # Used to create an Ingress record.
+  hosts:
+    - chart-example.local
```
Comment (alaa) [inline deploy/helm/device-registry/values.yaml]: can you change this to `device-registry.ota.local`

File: deploy/helm/device-registry/values.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: advancedtelematic/device-registry
+  tag: 0.0.1-22-gc286097
+  pullPolicy: IfNotPresent
+service:
+  name: device-registry
+  type: ClusterIP
```
Comment (alaa) [inline deploy/helm/device-registry/values.yaml]: why it's using ClusterIP here?

File: deploy/helm/device-registry/values.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  externalPort: 80
+  internalPort: 9001
+ingress:
+  enabled: false
```
Comment (alaa) [inline deploy/helm/device-registry/values.yaml]: is this suppose to be `true` in our case?

Comment (alexhumphreys) [reply deploy/helm/device-registry/values.yaml]: it's an internal service, so I didn't think it'd have an ingress. but i can change it to true for testing if that's what we want.

File: deploy/helm/device-registry/templates/service.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "device-registry.fullname" . }}
```
Comment (alaa) [inline deploy/helm/device-registry/templates/service.yaml]: seems like `.name` works as we expect.. not sure what's the point of `.fullname`

**Comment by alexhumphreys** [comment]
Superseded by ota-charts repository.

**Pull request closed without merging**